### PR TITLE
Service/ACT: move ACT services to folder

### PR DESCRIPTION
**Pull request by mailwl**

### CODE REVIEW

File: src/core/hle/service/act/act.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2016 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#pragma once
+
+namespace Service {
+
+class Interface;
```
Comment (bunnei) [inline src/core/hle/service/act/act.h]: Is this forward declaration used at all?

Comment (mailwl) [inline src/core/hle/service/act/act.h]: ATM no, but later i want to stub some functions in next PR, there is needed
https://github.com/mailwl/citra/blob/account-stub/src/core/hle/service/act/act.h

File: src/core/hle/service/act/act.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace Service {
+namespace ACT {
+
+void Init() {
+    AddService(new ACT_A);
+    AddService(new ACT_U);
+}
+
+void Shutdown() {}
```
Comment (yuriks) [inline src/core/hle/service/act/act.cpp]: Leave the shutdown function out if it does nothing.

File: src/core/hle/service/act/act.h (hunk 1 above)
Comment (yuriks) [inline src/core/hle/service/act/act.h]: Can you just add it in the next PR? This makes git blames, etc. kinda confusing.

**Comment by mailwl** [comment]
done

**Pull request merged**